src/psyclone/psyir/symbols/containersymbol.py

```diff
@@ -213,43 +212,36 @@ class FortranModuleInterface(ContainerSymbolInterface):
 
     @staticmethod
     def import_container(name):
```

Comment:
Not this PRs fault, but I wonder if there isn't a better name? `import_container`  does not really tell me what this function does (esp. not that it returns something, I would expect that something is added to a symbol table??).
Maybe `get_container_psyir`?

Reply:
Yeah, I like the new proposed name

Reply:
I've actually gone with `get_container`: since we're in the PSyIR here, I felt the `_psyir` was superfluous. Is that OK @sergisiso and @hiker?

Reply:
I'm happy with that. In the context of PSyclone, Container makes it obvious what it is.